runtime/src/main/java/org/corfudb/runtime/view/stream/StreamAddressSpace.java

```diff
@@ -94,32 +115,32 @@ private void mergeFrom(StreamAddressSpace other) {
      */
     public Long getTail() {
         // If no address is present for this stream, the tail is given by the trim mark (last trimmed address)
-        if (addressMap.isEmpty()) {
+        if (bitmap.isEmpty()) {
             return trimMark;
         }
 
         // The stream tail is the max address present in the stream's address map
-        return addressMap.getReverseLongIterator().next();
+        return bitmap.getReverseLongIterator().next();
     }
 
     /**
      * Add an address to this address space.
      *
      * @param address address to add.
      */
-    public void addAddress(long address) {
-        addressMap.addLong(address);
+    public void addAddress(Long address) {
```

Comment:
int to long implicit conversion should be safe. Revert type to a primitive long to prevent auto-boxing